### PR TITLE
Fix/638/remove job selection

### PR DESCRIPTION
**Pull request by superstar54**
This PR fixs #813 .

- Removed the workchain selector on the top
- Remove the `previous`, `next` and `reset` header.

Then, I added
- A new `Job list` button on the top
- A button to create a new calculation. It will create a new QE App page, instead of reset the current APP.

https://github.com/user-attachments/assets/ef821b62-4119-4659-9dc5-bb1c25414315

### CODE REVIEW

File: src/aiidalab_qe/app/main.py [hunk 1]
```diff
@@ -64,23 +68,26 @@ def __init__(self, qe_auto_setup=True):
...
+        # hide the header
+        self._wizard_app_widget.children[0].layout.display = "none"
         self._wizard_app_widget.observe(self._observe_selected_index, "selected_index")
 
-        # Add process selection header
-        self.work_chain_selector = QeAppWorkChainSelector(
-            layout=ipw.Layout(width="auto")
+        # Add a button to start a new calculation
+        self.new_work_chains_button = ipw.Button(
+            description="New calculation",
```
Comment (AndresOrtegaGuerrero) [inline src/aiidalab_qe/app/main.py:77]: `Start New Calculation`
Might be more explicit for the user

File: src/aiidalab_qe/app/utils/search_jobs.py [hunk 2]
```diff
@@ -70,6 +72,8 @@ def load_data(self):
         ]
 
     def setup_widgets(self):
+        import ipywidgets as ipw
```
Comment (AndresOrtegaGuerrero) [inline src/aiidalab_qe/app/utils/search_jobs.py:75]: why not to keep some of these import on top ? or somehow you expect better performance like this ?

**Comment by AndresOrtegaGuerrero** [comment]
I think the button named "Job list" it might not be intuitive to the users, Maybe we should re label to something more explicit? 
"Previous Calculations" or "View Previous Runs" or "Job History" , so we dont need some text to guide the user where to find the calculations

File: src/aiidalab_qe/app/utils/search_jobs.py (hunk 2 above)
Comment (superstar54) [reply src/aiidalab_qe/app/utils/search_jobs.py:75]: I only import the module when I use it—kind of lazy loading to speed the loading.

**Comment by AndresOrtegaGuerrero** [comment]
I think the same apply to the button Guide , maybe we should use 

"How to Use" or "Usage Guide" or even "Getting Started" , and maybe we can change the icon to `icon="fa-book",`

File: src/aiidalab_qe/app/main.py (hunk 1 above)
Comment (superstar54) [reply src/aiidalab_qe/app/main.py]: Thanks. I changed it to `Start New Calculation`.

**Comment by superstar54** [comment]
> I think the button named "Job list" it might not be intuitive to the users, Maybe we should re label to something more explicit?

Thanks for the suggestion. I changed the `Job list` to `Job History`. 

> "How to Use" or "Usage Guide" or even "Getting Started" , and maybe we can change the icon to icon="fa-book",

For the `Guide` button, I will let @edan-bainglass decide and make changes if needed.

**Review by AndresOrtegaGuerrero — APPROVED** [review]
LGTM! thank you Xing

**Comment by edan-bainglass** [comment]
> I think the same apply to the button Guide , maybe we should use\n\n\"How to Use\" or \"Usage Guide\" or even \"Getting Started\" , and maybe we can change the icon to icon=\"fa-book\",

I like Getting Started, and the book icon might be a nice touch. Maybe check what others do along Getting Started.

**Comment by AndresOrtegaGuerrero** [comment]
> > I think the same apply to the button Guide , maybe we should use\n\n"How to Use" or "Usage Guide" or even "Getting Started" , and maybe we can change the icon to icon="fa-book",
> 
> I like Getting Started, and the book icon might be a nice touch. Maybe check what others do along Getting Started.

Should we open an issue to replace it ? @superstar54

**Pull request merged**